stdlib/LinearAlgebra/src/transpose.jl

```diff
@@ -210,3 +210,14 @@ function copy_similar(A::TransposeAbsMat, ::Type{T}) where {T}
     C = similar(A, T, size(A))
     transpose!(C, parent(A))
 end
+
+function Base.copyto_unaliased!(deststyle::IndexStyle, dest::AbstractMatrix, srcstyle::IndexCartesian, src::AdjOrTransAbsMat)
+    if axes(dest) == axes(src)
+        f! = inplace_adj_or_trans(src)
+        f!(dest, parent(src))
+    else
+        invoke(Base.copyto_unaliased!, Tuple{IndexStyle, AbstractArray, IndexStyle, AbstractArray},
+            deststyle, dest, srcstyle, src)
```

Comment:
Style suggestion:

```suggestion
        @invoke Base.copyto_unaliased!(deststyle::IndexStyle, dest::AbstractArray, srcstyle::IndexStyle, src::AbstractArray)
```

I find this much easier to read, and it expands to exactly what you wrote.